sdk/src/main/java/com/qonversion/android/sdk/internal/QonversionRepository.kt

```diff
@@ -85,7 +86,11 @@ internal class QonversionRepository internal constructor(
                 if (body == null) {
                     callback.onError(errorMapper.getErrorFromResponse(it))
                 } else {
-                    callback.onSuccess(body)
+                    if (body.payload.isEmpty() && body.source == null) {
```

Comment:
It turns out that for clients configuration can't be null, but we made it nullable for parsing purposes

Reply:
We can make nullable field internal and add a public non nullable getter